README.md

```diff
@@ -168,6 +168,11 @@ sys.exit(app.exec_())
 
 Please note, for maximum compatibility, only pass the argument of the filename to the `load_ui` function.
 
+**sip API v2**
+
+If you're using PyQt4, `sip` attempts to set its API to version 2 for the following:
+- `QString`
+- `QVariant`
```

Comment:
Best to add all api's we're setting, less we're bound to return here at some point after an issue has been filed about it.